package.json

```diff
@@ -98,8 +98,7 @@
     "shelljs-nodecli": "~0.1.0",
     "sinon": "^1.17.2",
     "temp": "^0.8.3",
```

Comment:
Looked through the docs for both [`tmp`](https://github.com/raszi/node-tmp) and [`temp`](https://github.com/bruce/node-temp). They [both](https://github.com/raszi/node-tmp#graceful-cleanup) [facilitate](https://github.com/bruce/node-temp#want-cleanup-make-sure-you-ask-for-it) cleaning up after themselves, but we'd only instructed `temp` to do so by [calling `tmp.track()`](https://github.com/MethodGrab/eslint/blob/cleanup-test-temp-files/tests/lib/config/config-file.js#L26). Removing redundant dependency sounds good in my book. :+1: